src/rgw/rgw_lua_utils.cc

```diff
@@ -41,4 +41,23 @@ void stack_dump(lua_State* L) {
   std::cout << "--------------- Stack Dump Finished ---------------" << std::endl;
 }
 
+void set_package_path(lua_State* L, const std::string& install_dir) {
+  if (install_dir.empty()) {
+    return;
+  }
+  const auto path = install_dir+"/share/lua/5.3/?.lua";  
```

Comment:
might want to replace magic numbers with version numbers detected using `find_package()`.